analytic_tag_default/views/account_analytic_default.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_account_analytic_default_tree" model="ir.ui.view">
+        <field name="name">account.analytic.default.tree.inherit</field>
+        <field name="model">account.analytic.default</field>
+        <field name="inherit_id" ref="account_analytic_default.view_account_analytic_default_tree" />
+        <field name="arch" type="xml">
+            <field name="analytic_id" position="after">
+                <field name="tag_ids" groups="analytic.group_analytic_accounting" />
+            </field>
+        </field>
+    </record>
+
+    <record id="view_account_analytic_default_form" model="ir.ui.view">
+        <field name="name">account.analytic.default.form</field>
+        <field name="model">account.analytic.default</field>
+        <field name="inherit_id" ref="account_analytic_default.view_account_analytic_default_form" />
+        <field name="arch" type="xml">
+            <field name="analytic_id" position="attributes">
+                <attribute name="required">0</attribute>
+            </field>
+            <field name="analytic_id" position="after">
+                <field name="tag_ids" widget="many2many_tags" groups="analytic.group_analytic_accounting" />
```

Comment:
I have not see that the group is the same on the 2 fields.
But in this case, if you don't have the group, you never can create an `account.analytic.default` object with your constrains, because you can't define one of this 2 fields.